activejob/lib/active_job/exceptions.rb

```diff
@@ -24,7 +24,7 @@ module ClassMethods
       # ==== Options
       # * <tt>:wait</tt> - Re-enqueues the job with a delay specified either in seconds (default: 3 seconds),
       #   as a computing proc that takes the number of executions so far as an argument, or as a symbol reference of
-      #   <tt>:exponentially_longer</tt>, which applies the wait algorithm of <tt>((executions**4) + (Kernel.rand * (executions**4) * jitter)) + 2</tt>
+      #   <tt>:with_polynomial_backoff</tt>, which applies the wait algorithm of <tt>((executions**4) + (Kernel.rand * (executions**4) * jitter)) + 2</tt>
```

Comment:
`:polynomially_longer` would be a better name for this.

Reply:
Done in https://github.com/rails/rails/pull/49292/commits/3240060eb28b2af52fd36c69bb745c030c05a473